package.json

```diff
@@ -66,12 +67,12 @@
 		"karma-spec-reporter": "0.0.32",
 		"karma-webpack": "^4.0.2",
 		"minimatch": "^3.0.4",
-		"puppeteer": "^4.0.1",
 		"sade": "^1.7.3",
 		"script-loader": "^0.7.2",
 		"simple-code-frame": "^1.0.0"
 	},
 	"peerDependencies": {
+		"puppeteer": "^4.0.1",
```

Comment:
We could probably relax this a bit and do something like ">=2" I think. May be able to do "*" since karma-chrome-launcher uses the `chrome.exe` directly I think and doesn't use the puppeteer API directly

Reply:
oh good point 👍